lib/openshift/network_attachment_definition.rb

```diff
@@ -0,0 +1,9 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # represents an Object
+  class NetworkAttachmentDefinition < ClusterResource
+    RESOURCE = 'networkattachmentdefinition.k8s.cni.cncf.io'
```

Comment:
I'd guess that there should be an `s` in `networkattachmentdefinitions`. Could you paste an `oc get -o yaml` example to be sure?

Reply:
Actually I see now your  example output. It seems to be in fact `network-attachment-definitions`.

Reply:
under `selfLink`

Reply:
yep there is 's' but it works both ways 

apiVersion: k8s.cni.cncf.io/v1
kind: NetworkAttachmentDefinition
metadata:
creationTimestamp: "2019-11-12T00:20:43Z"
generation: 1
name: macvlan-bridge
namespace: test
resourceVersion: "190189"
selfLink: /apis/k8s.cni.cncf.io/v1/namespaces/test/network-attachment-definitions/macvlan-bridge
uid: fd2946d2-3608-4fa1-b174-52cd944ea057
spec:
config: '{ "cniVersion": "0.3.0", "type": "macvlan", "master": "eth0", "mode": "bridge",
"ipam": { "type": "host-local", "subnet": "10.1.1.0/24", "rangeStart": "10.1.1.100",
"rangeEnd": "10.1.1.200", "routes": [ { "dst": "0.0.0.0/0" } ], "gateway": "10.1.1.1"
} }'